tests/AzureMcp.Tests.csproj

```diff
@@ -12,6 +12,7 @@
   </PropertyGroup>
   
   <ItemGroup>
+    <PackageReference Include="Azure.Monitor.Ingestion" />
```

Comment:
Just curious if we need to use the ingestion package to send logs or is there a way to send the logs through OTel?

Reply:
I'm not sure, do you know?

Reply:
tagging @lmolkova to see if we can use .NET OTel lib to send the logs to Azure Monitor.

Reply:
just keep using ILogger and just enable otel - it'll forward all ilogger to AzMon, no need for low-level Azure.Monitor.Ingestion

Reply:
We went with the Ingestion APIs because we need a custom table, which the Logger/Otel approach doesn't seem to support. Since this is working as-is and the suggestion is an improvement, I'm going to create this issue and get this fix in.

https://github.com/Azure/azure-mcp/issues/290